nf_core/pipeline-template/assets/methods_description_template.yml

```diff
@@ -0,0 +1,25 @@
+id: "${workflow.manifest.name.replace('/', '-')}-methods-description"
+description: "Suggested methods description text of pipeline usage"
```

Comment:
```suggestion
description: "Suggested methods description text of pipeline usage."
```
This solves the duplication issue

Reply:
Whaaaaat? Ok... That's strange. But thank you very much!

Reply:
This smells like a bug. I have a vague feeling that it may have been fixed already? I think it was something to do with the new DOI thing in MultiQC.

Reply:
Ok - I was running with 1.12 when testing so could be.